pkg/server/service.go

```diff
@@ -243,11 +245,12 @@ func (c *criContainerdService) Stop() {
 
 // getDeviceUUID gets device uuid for a given path.
 func (c *criContainerdService) getDeviceUUID(path string) (string, error) {
-	info, err := c.os.LookupMount(path)
+	mount, err := c.os.LookupMount(path)
```

Comment:
`LookupMount` has a comment in it:
> 		// Note that m.{Major, Minor} are generally unreliable for our purpose here
>		// https://www.spinics.net/lists/linux-btrfs/msg58908.html

But we are now using exactly those fields below.

Reply:
Actually, we don't seem to rely on these any more or less than we did before, so I guess this is fine.

Reply:
@Random-Liu @ijc Interesting.. if "generally unreliable" maybe we should do it both ways and compare results then return the best result.  Just thinking out loud and now worrying about what was meant by generally unreliable.